treeherder/perf/alerts.py

```diff
@@ -60,7 +60,9 @@ def generate_new_alerts_in_series(signature):
     # (2) the alerts max age
     # (use whichever is newer)
     max_alert_age = datetime.now() - settings.PERFHERDER_ALERTS_MAX_AGE
-    series = PerformanceDatum.objects.filter(signature=signature, push_timestamp__gte=max_alert_age)
+    series = PerformanceDatum.objects.filter(
+        signature=signature, push_timestamp__gte=max_alert_age
+    ).prefetch_related("performancedatumreplicate_set")
```

Comment:
Can you split this into two queries? I'd rather not impact the existing queries for performance data being used for alerts.